src/libraries/Microsoft.Extensions.Configuration.Json/src/JsonConfigurationFileParser.cs

```diff
@@ -14,7 +13,7 @@ internal class JsonConfigurationFileParser
         private JsonConfigurationFileParser() { }
 
         private readonly IDictionary<string, string> _data = new SortedDictionary<string, string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
While you're here..?
```suggestion
        private readonly SortedDictionary<string, string> _data = new SortedDictionary<string, string>(StringComparer.OrdinalIgnoreCase);
```